src/components/TopBar/CallTime.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+  - @copyright Copyright (c) 2022 Marco Ambrosini <marcoambrosini@icloud.com>  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<NcPopover class="top-bar__button call-time"
+		close-after-click="true"
+		:menu-title="callTime"
+		:shown.sync="showPopover"
+		:class="{ 'call-time--wide': isWide }"
+		:triggers="[]"
+		:container="container">
+		<template #trigger>
+			<NcButton :disabled="!isRecording"
+				:wide="true"
+				:class="{ 'call-time__not-recording': !isRecording }"
+				type="tertiary"
+				@click="showPopover = true">
+				<template v-if="isRecording" #icon>
+					<RecordCircle :size="20"
+						fill-color="#e9322d" />
+				</template>
+				{{ formattedTime }}
+			</ncbutton>
+		</template>
+		<NcButton type="tertiary-no-background"
+			:wide="true"
+			@click="stopRecording">
+			<template #icon>
+				<StopIcon :size="20" />
+			</template>
+			{{ t('spreed', 'Stop recording') }}
```

Comment:
While the options in the NcActions are grey/max contrast/something
This one is pretty dark/black or white/bright:
Before | After
---|---
![Bildschirmfoto vom 2022-11-21 11-49-19](https://user-images.githubusercontent.com/213943/203032088-447b1b7d-a9a4-4fb6-b2bd-d0e7111f3c60.png) | ![Bildschirmfoto vom 2022-11-21 11-49-25](https://user-images.githubusercontent.com/213943/203032089-c2ccfe08-3f7f-4e7f-a1d9-cc512384fcbe.png)
![Bildschirmfoto vom 2022-11-21 11-51-13](https://user-images.githubusercontent.com/213943/203032273-ea6ee968-e7ab-464d-8446-6023dd18d0a8.png) | ![Bildschirmfoto vom 2022-11-21 11-51-18](https://user-images.githubusercontent.com/213943/203032278-1e878ca0-d8f8-4c00-9958-0a165ca6cfa8.png)

Also the rounding is different from the actions menu, but we can move this to a follow up, as the PR has too many things touching already.

Reply:
https://github.com/nextcloud/spreed/issues/8388